src/sas/data_explorer_error_message.py

```diff
@@ -0,0 +1,17 @@
+from PySide6.QtWidgets import QMessageBox, QWidget
+
+
+class DataExplorerErrorMessage(QMessageBox):
+    def __init__(self, parent: QWidget, errors: list[ValueError]):
+        super().__init__(parent)
+        self.setIcon(QMessageBox.Icon.Critical)
+        self.setWindowTitle('Data Error')
+        if len(errors) == 1:
+            error = errors[0]
+            self.setText(str(error))
+        else:
+            full_error_msg = "Some data could not be removed due to the following errors:\n"
+            error_msg_set = set([str(err) for err in errors])
+            error_msg_str = '\n'.join(error_msg_set)
+            full_error_msg += f"\n{error_msg_str}"
```

Comment:
Optional suggestion: instead of using a `set`, we could use a `defaultdict` that starts at zero and keep track of the number of instances of each error.  This way we can inform the user of multiple occurances of an error.

I'm thinking of a user who sees that they see that error X occured, fixes error X, but then error X goes away because there's 300 instances of it and they have only fixed one.  The count would at least show them that the count has decremented.